docs/index.md

```diff
@@ -1,34 +1,39 @@
-# Home
+# DIVE Documentation
 
-The documentation for VIAME DIVE, an open-source AI system for analyzing imagery and video.
+This is the documentation for DIVE, an open-source annotation system that integrates with VIAME.
 
-See the full ecosystem of tools at [viametoolkit.org](https://www.viametoolkit.org/)
+## Definitions
 
-Need help?
+* **DIVE** is the annotator and data management software system.  It is our name for the code and capabilities, including both web and desktop, that can be deployed and configured for a variety of needs.  It integrates tightly with the VIAME, but is separate from that tool suite.
```

Comment:
`It integrates tightly with **the** VIAME`  

Nowhere else is it referred to as 'the' VIAME